evm/test/libraries/IntegrationHelpers.sol

```diff
@@ -0,0 +1,197 @@
+import "../../src/interfaces/IWormholeTransceiver.sol";
+import {WormholeRelayerBasicTest} from "wormhole-solidity-sdk/testing/WormholeRelayerTest.sol";
+import "../../src/NttManager/NttManager.sol";
+import "wormhole-solidity-sdk/testing/helpers/WormholeSimulator.sol";
+
+contract IntegrationHelpers is Test {
+    using TrimmedAmountLib for uint256;
+    using TrimmedAmountLib for TrimmedAmount;
+
+    WormholeTransceiver wormholeTransceiverChain1;
+    WormholeTransceiver wormholeTransceiverChain1Other;
+    WormholeTransceiver wormholeTransceiverChain2;
+    WormholeTransceiver wormholeTransceiverChain2Other;
+
+    function buildTransceiverInstruction(bool relayer_off)
+        public
+        view
+        returns (TransceiverStructs.TransceiverInstruction memory)
+    {
+        WormholeTransceiver.WormholeTransceiverInstruction memory instruction =
+            IWormholeTransceiver.WormholeTransceiverInstruction(relayer_off);
+
+        bytes memory encodedInstructionWormhole;
+        // Source fork has id 0 and corresponds to chain 1
+        if (vm.activeFork() == 0) {
+            encodedInstructionWormhole =
+                wormholeTransceiverChain1.encodeWormholeTransceiverInstruction(instruction);
+        } else {
+            encodedInstructionWormhole =
+                wormholeTransceiverChain2.encodeWormholeTransceiverInstruction(instruction);
+        }
+        return TransceiverStructs.TransceiverInstruction({
+            index: 0,
+            payload: encodedInstructionWormhole
+        });
+    }
+
+    function encodeTransceiverInstruction(bool relayer_off) public view returns (bytes memory) {
+        TransceiverStructs.TransceiverInstruction memory TransceiverInstruction =
+            buildTransceiverInstruction(relayer_off);
+        TransceiverStructs.TransceiverInstruction[] memory TransceiverInstructions =
+            new TransceiverStructs.TransceiverInstruction[](1);
+        TransceiverInstructions[0] = TransceiverInstruction;
+        return TransceiverStructs.encodeTransceiverInstructions(TransceiverInstructions);
+    }
+
+    function _setTransceiverPeers(
+        WormholeTransceiver[2] memory transceivers,
+        WormholeTransceiver[2] memory transceiverPeers,
+        uint16[2] memory chainIds
+    ) internal {
+        for (uint256 i; i < transceivers.length; i++) {
+            transceivers[i].setWormholePeer(
+                chainIds[i], toWormholeFormat(address(transceiverPeers[i]))
+            );
+        }
+    }
+
+    function _setManagerPeer(
+        NttManager sourceManager,
+        NttManager peerManagerAddr,
+        uint16 peerChainId,
+        uint8 peerDecimals,
+        uint64 inboundLimit
+    ) internal {
+        sourceManager.setPeer(
+            peerChainId, toWormholeFormat(address(peerManagerAddr)), peerDecimals, inboundLimit
+        );
+    }
+
+    function _enableSR(WormholeTransceiver[2] memory transceivers, uint16 chainId) internal {
```

Comment:
Should future proof this by naming it `enableEvmSR`.

Reply:
this method passes in an arbitrary chainId - more future proof with the current name imo